src/components/Editor.jsx

```diff
@@ -114,6 +118,14 @@ class Editor extends React.Component {
     }
   }
 
+  _toggleEditorTextSize(textSizeIsLarge) {
```

Comment:
This method name is a little misleading, as it sounds like it’s something to do with changing the current state of the font size rather than just syncing the editor with the underlying state. I would call it e.g. `_applyFontSize` (also no need for `Editor` as we are already in the `Editor` component)